packages/tupaia-web/src/api/queries/useFetchMapOverlays.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { useQuery } from 'react-query';
+import { get } from '../api';
+import { EntityCode, ProjectCode } from '../../types';
+
+export const useFetchMapOverlays = (projectCode?: ProjectCode, entityCode?: EntityCode) => {
```

Comment:
I think it would be good to just move all the code from packages/tupaia-web/src/utils/useMapOverlays.ts into this file since it is just derived api data with the exception of `updateSelectedMapOverlay`. You could then put updateSelectedMapOverlay inline in  MapOverlayList, make it a stand alone util or keep it with useMapOverlays if you really want.